openedx/adg/lms/applications/helpers.py

```diff
@@ -20,3 +22,60 @@ def validate_logo_size(file_):
     size = getattr(file_, 'size', None)
     if size and LOGO_IMAGE_MAX_SIZE < size:
         raise ValidationError(_('File size must not exceed {size} KB').format(size=LOGO_IMAGE_MAX_SIZE / 1024))
+
+
+# pylint: disable=translation-of-non-string
+def check_validations_for_past_record(attrs, error_message, key_completed_date_month, key_completed_date_year):
```

Comment:
I think we can avoid passing keys for completed date month and completed date year and use keys directly. just like you are using for start date year and start date month in the same method.